tests/__init__.py

```diff
@@ -21,3 +21,5 @@
 
 # run tests with C locales
 os.environ["LC_ALL"] = "C"
```

Comment:
Can you please change this to `C.UTF-8`?

Reply:
Done :)